src/Altinn.AccessManagement.Core/Models/Rights/Right.cs

```diff
@@ -15,11 +16,13 @@ public class Right
         /// <summary>
         /// Gets or sets the list of resource matches which uniquely identifies the resource this right applies to.
         /// </summary>
+        [Required]
```

Comment:
Not sure whats happened where the internal models shouldn't need data annotations for serialization

Reply:
Fixed. Also fixed some missing annotations on the external models